app/models/user.py

```diff
@@ -170,6 +170,7 @@ def generate_fake(count=100, **kwargs):
                 email=fake.email(),
                 password='password',
                 confirmed=True,
+                approved=False,
```

Comment:
approved is by default false, but thanks for the addition -- this was what i was asking about on slack